akka-docs-dev/rst/scala/code/docs/stream/ReactiveStreamsDocSpec.scala

```diff
@@ -135,4 +133,16 @@ class ReactiveStreamsDocSpec extends AkkaSpec {
     assertResult(storage)
   }
 
+  "use a processor" in {
+
+    //#use-processor
+    // An example Processor factory
+    def createProcessor: Processor[Int, Int] = Flow[Int].toProcessor.run()
+
+    val src: Source[Int, Unit] = Source.single(42).viaProcessor(() => createProcessor)
+    val flow: Flow[Int, Int, Unit] = Flow[Int].viaProcessor(() => createProcessor)
```

Comment:
Perhaps I am unclear: I am -extremely against- `viaProcessor` as -nearly every single method on Flow- -semantically- is *via* but doesn't carry it in its name.

I'm not married to the idea of calling it `process` but the `via` part of that suggestion is only inconsistent.